integrations/google_ai/src/haystack_integrations/components/generators/google_ai/chat/gemini.py

```diff
@@ -326,12 +326,12 @@ def _get_response(self, response_body: GenerateContentResponse) -> List[ChatMess
         for candidate in response_body.candidates:
             for part in candidate.content.parts:
                 if part.text != "":
-                    replies.append(ChatMessage.from_system(part.text))
+                    replies.append(ChatMessage.from_assistant(part.text))
                 elif part.function_call is not None:
                     replies.append(
                         ChatMessage(
                             content=dict(part.function_call.args.items()),
-                            role=ChatRole.SYSTEM,
+                            role=ChatRole.ASSISTANT,
```

Comment:
@anakin87 Should function_call be assigned `assistant` or `system` role?

Reply:
`assistant`